core/src/style/polygonStyle.cpp

```diff
@@ -65,56 +96,114 @@ void PolygonStyle::constructShaderProgram() {
     m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
 }
 
-VboMesh* PolygonStyle::newMesh() const {
-    return new Mesh(m_vertexLayout, m_drawMode);
-}
+namespace {
```

Comment:
Yeah, saw that too - what do you suggest? Using named namespace from filename (minus 'Style') or prefix all structs with StyleX ?